<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="loginGroupBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="comboBoxMajors">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(192, 191, 188);</string>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Majors</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxCatalog">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Catalog Year</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameText">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>30</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cwidText">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>70</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>CWID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submitButton">
     <property name="geometry">
      <rect>
       <x>127</x>
       <y>170</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="errorGroupBox">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>60</y>
       <width>190</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="errorLabel">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>Please complete all boxes</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
     <widget class="QPushButton" name="errorSubmit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Understand</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 120, 0);
font: 700 24pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Virtual Advisor</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="resultsGroupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>661</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
   </widget>
   <zorder>resultsGroupBox</zorder>
   <zorder>loginGroupBox</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #00224b;
font: 11pt &quot;Ubuntu&quot;;
color: white;
</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionCopy"/>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
